src/web/demo.rkt

```diff
@@ -546,4 +559,4 @@
    (gen-file-not-found-responder (web-resource "404.html"))))
 
 (module+ main
-  (run-demo #t))
+  (run-demo #t))
```

Comment:
Please leave the terminal newline. Your editor likely has a preference thing for it.